### PR TITLE
Added the minimum and maximum caps for recommended calorie intake.

### PR DESCRIPTION
**Pull request by yuqiaoluolong**

### CODE REVIEW

File: src/main/java/seedu/dietbook/calculator/Calculator.java [hunk 1]
```diff
@@ -319,6 +319,11 @@ public int calculateRecomendation(Person person) {
         } else {
             recomendation = (int) requirement + 100;
         }
+        if (recomendation < 1000) {
```
Comment (HengFuYuen) [inline src/main/java/seedu/dietbook/calculator/Calculator.java]: Perhaps consider refactoring to avoid long methods.

**Review by HengFuYuen — COMMENTED** [review]
Perhaps some changes can be made to improve code quality and you can consider adding JUnit tests to test the newest changes.

File: src/main/java/seedu/dietbook/calculator/Calculator.java (hunk 1 above)
Comment (yuqiaoluolong) [reply src/main/java/seedu/dietbook/calculator/Calculator.java]: OK, I see.

**Review by HengFuYuen — APPROVED** [review]
Current commit LGTM

**Pull request merged**